src/common/view-definitions/version-3/split-definition.ts

```diff
@@ -62,6 +62,7 @@ const numberSplitConversion: SplitDefinitionConversion<NumberSplitDefinition> =
   toSplitCombine(split: NumberSplitDefinition): Split {
     const { dimension, limit, sort, granularity } = split;
     return new Split({
+      type: SplitType.number,
```

Comment:
Is it feasible to enhance split definition tests to cover this change?

Reply:
It's covered via viewDefinition converter, which are failing right now. I'm on it.

Reply:
OTOH, I'm thinking about overriding Split constructor, so you need provide all fields - no default values.